### DISCUSSION TITLE
How to get total documents indexed?

### DISCUSSION THREAD

**Discussion by d47081** [manticoresoftware/manticoresearch-php#164] [Q&A] [answered]
Can't find a method to output documents total stored in index

I just want to make statistic info in application without unlimited search query running.

e.g. documents total: 10000

**Comment by sanikolaev** [comment, marked as answer]
```php
<?php
require_once __DIR__ . '/vendor/autoload.php';

$config = ['host'=>'127.0.0.1','port'=>9308];
$client = new \Manticoresearch\Client($config);
$index = $client->index('t');
$index->drop(true);
$index->create(['a' => ['type' => 'int']]);
$docs = []; for ($n=0;$n<100000;$n++) $docs[] = ['id' => 0, 'a' => rand(0,1000)];
$index->addDocuments($docs);
$result = $index->search('')->option('cutoff', 0)->limit(0)->get()->getTotal();
echo $result."\n";
```

```
# php test_count.php
100000
```

**Reply by d47081** [reply]
Thank you for reply, I think about same implementation

just asked because of maybe some method exists like `$index->total()` that return meta info without search query construction

(to reduce the server load)

**Reply by sanikolaev** [reply]
It can be done using the `show table <name> status`, e.g.:

```
# php test_count.php
100000

# mysql -P9306 -h0 -e "show table t status"|grep indexed_documents
indexed_documents	100000
```

The count may be not as accurate as via `select` though. You can do it using the [sql](https://github.com/manticoresoftware/manticoresearch-php/blob/master/docs/sql.md) method in the PHP client.
